src/Symfony/Component/Translation/Tests/DependencyInjection/TranslationPathsPassTest.php

```diff
@@ -37,7 +37,7 @@ public function testProcess()
             ->setArguments([null, null, null, null, null, null, [], []])
         ;
         $container->register(ControllerArguments::class, ControllerArguments::class)
-            ->setTags(['controller.service_arguments'])
+            ->setTags(['controller.service_arguments' => []])
```

Comment:
It seems to me to be a breaking change if people inject random values via 
`Definition::setTags()` and expect to retrieve these values with `Definition::getTags()`.